README.md

```diff
@@ -1,3 +1,10 @@
 # NodeJS-MongoDB-Demo
 
-This is an example app that work with mongodb on clever cloud.
\ No newline at end of file
+This is an example NodeJS application that works with MongoDB on Clever Cloud.
+
+The application is a very simple list where you can add or delete values.
+
+To install it, simply fork this repository and create an application from your GitHub repo.
+Then create a MongoDB add-on and link it to your application.
+
+That's it, the application will use the environnement variable to connect to MongoDB
```

Comment:
variables* and there's a missing dot at the end :D 

(when I don't have anything else to correct, I tend to nitpick quite a bit)

Reply:
C'est mis à jour

Reply:
:p